### PR TITLE
feat: added avatar and badge component (P0)

### PR DESCRIPTION
**Pull request by shraddha15-creator**

### CODE REVIEW

**Comment by shraddha15-creator** [comment]
Added: Avatar & Badge component. 

avatar
badge

**Review by NtshVrm — APPROVED** [review]
Looks good👏

**Review by ayush65 — APPROVED** [review]
Looks good

**Review by ScoobyDubeyDo — APPROVED** [review]

**Comment by devdeadviz** [comment]
Add a short description about each variant, like how a user can use that in his/her own codebase.

File: pages/components/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (devdeadviz) [inline pages/components/avatar/avatar.html:6]: Suggestion: You can add your name in meta tags, this gives a flavor of uniqueness and owing codes.

File: pages/components/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		<script
+			src="https://kit.fontawesome.com/f9a7380946.js"
+			crossorigin="anonymous"
+		></script>
+		<link rel="stylesheet" href="../../../style.css" />
+		<link rel="stylesheet" href="../../docs.css" />
+		<link rel="stylesheet" href="./avatar.css" />
```
Comment (devdeadviz) [inline pages/components/avatar/avatar.html:13]: Try to import only a single stylesheet in an HTML and if you require styles from other stylesheets add them in your that stylesheet that you have imported in your HTML using import.

**Review by devdeadviz — COMMENTED** [review]
Run prettier once in your HTML files, it will fix the indentation problem. Rest looks good to me,

**Pull request merged**